lte/gateway/c/session_manager/SetMessageManagerHandler.cpp

```diff
@@ -88,6 +90,22 @@ SessionConfig SetMessageManagerHandler::m5g_build_session_config(
   return cfg;
 }
 
+bool SetMessageManagerHandler::validate_session_request(

```

Comment:
why do you need a separate function to check this?
It looks like you are creating the same error twice .
If you put it on the function you can use the same error for MLOG, events_reported and GRPC response

Reply:
Yes, handled with in a function.